frontend/src/state/codeSlice.ts

```diff
@@ -1,12 +1,9 @@
 import { createSlice } from "@reduxjs/toolkit";
-import { INode, flattenTree } from "react-accessible-treeview";
```

Comment:
I think we can remove `react-accessible-treeview` from dependencies. I forgot to clear this file after remaking the file explorer